pkg/operator/certrotationcontroller/certrotationcontroller.go

```diff
@@ -49,8 +62,8 @@ func NewCertRotationController(
 		certrotation.TargetRotation{
 			Namespace: operatorclient.OperatorNamespace,
 			Name:      "csr-signer",
-			Validity:  1 * 4 * time.Hour, // to be 5 days
-			Refresh:   2 * time.Hour,     // to be 1 day
+			Validity:  30 * rotationDay,
+			Refresh:   15 * rotationDay,
```

Comment:
any reason not to rotate this every day?

Reply:
> any reason not to rotate this every day?

oh, you're chasing the restart.  Someone look to see if we can just make it reload serving trust dynamically?

Reply:
Discussed on slack, I'll be looking into that dynamic reload of kcm certs in kas.